code/_onclick/hud/map_popups.dm

```diff
@@ -22,14 +22,14 @@
 		ASYNC
 			// HACK: This fixes the character creator in 516 being small and relying on other byondui things (like cameras) to open in order to update and refresh.
 			// This also will fix the camera console screen being offset, Gateway, and admin pod panel.
-			// Adding 100 then setting it back seemed to do the trick!
+			// Setting the alpha value to 254 then back to fixes it
 			// Why the fuck does this work? This is some byond bug and I honestly have no fucking clue why this works.
 			// I don't think plane master will be affected, I hope.
 			// We're stuck in the belly of this awful machine.
-			sleep(0.2 SECONDS)
-			screen_loc = "[assigned_map]:[x+100]:[px],[y+100]:[py]"
-			sleep(0.2 SECONDS)
-			screen_loc = "[assigned_map]:[x]:[px],[y]:[py]"
+			sleep(world.tick_lag)
+			alpha = 254
+			sleep(world.tick_lag)
+			alpha = 255
```

Comment:
does this work on its own? was concerned about rendering times or some such

Reply:
nope